presto-spark-base/src/main/java/com/facebook/presto/spark/execution/PrestoSparkOutputBuffer.java

```diff
@@ -79,6 +85,25 @@ public T get()
                 memoryManager.updateMemoryUsage(-rowBatch.getRetainedSizeInBytes());
             }
         }
+        if (rowBatch != null) {
```

Comment:
Is it possible for `rowBatch` to be Type `PrestoSparkBufferedSerializedPage`,  then we can just call it `batch`.

Reply:
`PrestoSparkBufferedSerializedPage` is technically still a batch of rows, just encoded in a columnar fashion